scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -345,9 +336,9 @@ def parse_args(o3de_context: exp.O3DEScriptExportContext):
                                            disable_override_arg=['-noregovr', '--disallow-registry-overrides'],
                                            disable_override_desc="Disallow overriding registry settings from external sources during the cmake build configuration.")
 
-        default_asset_bundling_path = export_config.get_value(exp.SETTINGS_DEFAULT_ASSET_BUNDLING_PATH.key, exp.SETTINGS_DEFAULT_ASSET_BUNDLING_PATH.default)
-        parser.add_argument('-abp', '--asset-bundling-path', type=pathlib.Path, default=pathlib.Path(default_asset_bundling_path),
-                            help=f"Designates where the artifacts from the asset bundling process will be written to before creation of the package. If not specified, default is {default_asset_bundling_path}.")
+        # Do not supply a default path for the asset bundling path. This should be inferred by the export script based on engine installation.
+        parser.add_argument('-abp', '--asset-bundling-path', type=pathlib.Path, default=None,
+                            help=f"Designates where the artifacts from the asset bundling process will be written to before creation of the package. If not specified, the path is inferred based on engine installation.")
```

Comment:
I thought this defaults to _<project_path>/build/asset_bundling_, regardless of engine SDK or engine from source.